Makefile

```diff
@@ -63,7 +64,19 @@ markdown-lint:
 
 .PHONY: test
 test:
-	@echo "testing placeholder"
+	@echo "Running tests in each solution example directory..."
+	@for dir in $(EXAMPLES_DIRS); do \
+		echo "Testing in $$dir"; \
+		$(MAKE) -C $$dir test; \
+	done
+
+.PHONY: integration_test
+integration_test:
+	@echo "Running local tests in each solution example directory..."
+	@for dir in $(EXAMPLES_DIRS); do \
```

Comment:
If we do this, we won't be able to parallel runs of these targets using `make -j`. I prefer creating the targets dynamically approach I had in my previous PR.

Reply:
Thanks for explaining the rationale, done.